### PR TITLE
Pull request for test functions for status and content update

### PR DESCRIPTION
**Pull request by tobuya**
-  Write tests for functions responsible for the status and content updates

### CODE REVIEW

File: src/modules/create.test.js [hunk 1]
```diff
@@ -34,4 +34,26 @@ describe('Add and delete tasks', () => {
...
+    todoList.addToList('Forth Test');
+    todoList.editTask(0, 'Fifth Test');
+    expect(todoList.tasks[0]?.description).toBe('Fifth Test');
+  });
+
+  test('Check status', () => {
+    todoList.completedTask(0, true);
+    expect(todoList.tasks.at(0).completed).toBe(true);
+  });
 });
```
Comment (ShadyShawkat) [inline src/modules/create.test.js:59]: - [ ] Please use mock for functions that use localStorage or DOM.
- [ ] Please also make sure to cover all edge cases of all of the functions.
   - As an example of covering all edge cases of `clearAllCompleted` function:
      - calling the function when there are no completed todos.
      - calling the function when there are completed todos.
      - calling the function when all todos are completed.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @tobuya, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/create.test.js (hunk 1 above)
Comment (tobuya) [reply src/modules/create.test.js:59]: @ShadyShawkat Thank you for the feedback. Do you have some suggestions? It will really help me since my partner left and I have no one to talk to on this. Thank you.

Comment (tobuya) [reply src/modules/create.test.js:59]: https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/tests_2.md#optional-requirements
Hey! This is an optional requirement for project approval. I would appreciate your help since my partner left and I have been working on this alone. I am also trying to finish my project on time.

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @tobuya, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights:
- All linters passing
- All requirements completed
- Professional readme

giphy

Cheers and Happy coding!👏👏👏

# Note: All mandatory requirements completed 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
